.github/workflows/planemo.yml

```diff
@@ -10,12 +16,19 @@ jobs:
       matrix:
         chunk: [1, 2, 3]
     steps:
-    - uses: actions/checkout@v1
-    - uses: "dpryan79/github-actions/@master"
+    - uses: actions/checkout@v3
+    - uses: mamba-org/setup-micromamba@main
+      with:
+        environment-file: .github/planemo.yml
+        cache-downloads: true
+        environment-name: foo
```

Comment:
can we name that differently, that might help in during debugging?

Reply:
I would not use a different yml file for planemo. We need to ensure its the same that we use for testing the unit-tests isn't it? We could just do `pip install planemo` after we installed deeptools maybe? Am I missing something?

Reply:
env renamed to 'test_and_build' (due to lack of inspiration :) ), and yes, you have a good point:
both planemo / pytests & builds now run from 1 environment.